src/components/GalleryPane/GalleryPane.js

```diff
@@ -106,31 +102,63 @@ export const GalleryPane = ({ galPhotos, profPhoto }) => {
           </div>
           <div className="gal-diamond g-24"></div>
           <div className="gal-diamond g-25"></div>
-          <div className="gal-diamond g-26">
-            {galPhotos[10] && <img 
+          <div className="gal-diamond g-26"></div>
+          <div className="gal-diamond g-27">
+          {galPhotos[1] && <img 
                                 alt="User photo" 
                                 className="gal-img"
-                                src={galPhotos[10]}
+                                src={galPhotos[1]}
                               />}
           </div>
-          <div className="gal-diamond g-27"></div>
           <div className="gal-diamond g-28">
-            {galPhotos[11] && <img 
+            {galPhotos[0] && <img 
+                                alt="User photo" 
+                                className="gal-img"
```

Comment:
Ah so this PR does some work on executors and the gallery both

Reply:
yep. I was on a design refactor roll.